test/validators.test.js

```diff
@@ -10819,6 +10819,34 @@ describe('Validators', () => {
     args: [],
   });
 
+  // he-IL non-Geographic
+  test({
```

Comment:
Please add the test set to https://github.com/jamro/validator.js/blob/86911d817fbcd177c295b13d7ca0fdff87341191/test/validators.test.js#L7252
with the proper locale